index.md

```diff
@@ -19,14 +19,31 @@ GET /jsonapi/node/article/{id}
 GET /jsonapi/node/article/{id}
 Accept-Language: fr
 
-200 OK
-
 // If translation exists:
+200 OK
 Content-Language: fr
 Content-Location: /jsonapi/node/article/{id}?lang=fr
```

Comment:
Is it ok to have the same resource available under multiple URLs? Or would this mean that a standalone *translation* resource is available at that URL?

Reply:
I think it's fine to have multiple URLs for the same resource.